### PR TITLE
Fixes more than one prize win in Orion Trail

### LINKED ISSUE #11401: Cheating Orion Trail (Enter spam edition)
watched a player do this and reproduced it a few times.  You can spam enter using some tab stuff at the end screen of the orion trail to have it spit out anywhere from 3-9 toys.

Steps to reproduce:
1. Beat the orion trail until you are on the screen where it says "Welcome to Orion" or whatever and you just need to press continue once more to have it spit out a toy
2. Press tab until you see the little outline over the continue button
3. Spam the fuck outta enter 
4. Watch it congratulate you like 8 times and spit out 8 toys too!
5. Repeat for INFINITE TOY GOODNESS OH LA LA

### PR DESCRIPTION
**Pull request by Jalleo**
fixes #11401
Not certain if it does though I need someone to test it to be certain my local machine is too quick

### CODE REVIEW

**Comment by MrStonedOne** [comment]
I mean, it does fix it, kinda but that's not how it should be fix.

This is still href hackable to get a prize. topic() needs a lot more checks

**Comment by Jalleo** [comment]
If ya willing to explain it to me on IRC I can patch it up I don't know anything about the href exploits

**Comment by MrStonedOne** [comment]
Basically: Just because a button isn't displayed on the clients browse window, doesn't mean they can't simulate clicking that button.

In browse html windows, you have to ensure they don't simulate clicks to buttons that aren't there, by ensuring that in your topic(), you check that they were suppose to be able to click that button.

Right now, what your change does, is try to ensure that the window reloads before it will accept another click, so that it can hide the win button. (busy is a hacky fix that just hides the issue). but nothing stops the user from preventing the window from reloading, then clicking it again, still getting more prizes.

I mean, topic() just flat out needs a massive rewrite. You can still exploit to win bombs over and over if you have an emag. I'm gonna go hop on sybil to confirm.

File: code/game/machinery/computer/arcade.dm [hunk 1]
```diff
@@ -466,6 +466,7 @@
 
 	if (href_list["continue"]) //Continue your travels
 		if(turns >= ORION_TRAIL_WINTURN)
+			busy = 1
```
Comment (Jalleo) [inline code/game/machinery/computer/arcade.dm]: changing to a better fix for duping extra stuff

**Comment by Jalleo** [comment]
@MrStonedOne better?

File: config/admins.txt [hunk 2]
```diff
@@ -92,3 +92,4 @@ Dunc = Game Master
 MMMiracles = Game Master
 bear1ake = Game Master
 CoreOverload = Game Master
+Jalleo = Game Master
```
Comment (Razharas) [inline config/admins.txt]: omg stealth admin status addition reported

Comment (Jalleo) [inline config/admins.txt]: Sorry forgot to report it was my second or third commit in

**Pull request merged**